controllers/index.js

```diff
@@ -0,0 +1,24 @@
+const CoffeeForm = require("../models/CoffeeForm");
+const asyncCatcher = require("../utils/asyncCatcher");
+const CoffeeFormService = require("../services/CoffeeFormService");
+// eslint-disable-next-line no-unused-vars
+
+const CoffeeFormInstance = new CoffeeFormService(CoffeeForm);
+// eslint-disable-next-line no-unused-vars
+exports.submit = asyncCatcher(async (req, res, next) => {
```

Comment:
```suggestion
/* eslint-disable no-unused-vars */
const CoffeeForm = require("../models/CoffeeForm");
const asyncCatcher = require("../utils/asyncCatcher");
const CoffeeFormService = require("../services/CoffeeFormService");

const CoffeeFormInstance = new CoffeeFormService(CoffeeForm);

exports.submit = asyncCatcher(async (req, res, next) => {
```

이렇게 최상단에 하나만 바꿔주시면 될 것 같습니다~